From: zeertzjq <zeertzjq@outlook.com>
Date: Fri, 5 Aug 2022 08:32:54 +0800
X-Dgit-Generated: 0.7.2-4 0f00e0dc8d316ea2fc0198e55b0077a7f61a3e69
Subject: test(let_spec): increase expect_exit() timeout

Test ":unlet self-referencing node in a List graph #6070" feeds many characters into typeahead, so a timeout of only 100 milliseconds sometimes fails. Change that timeout to 1000 milliseconds.

---

diff --git a/test/functional/vimscript/let_spec.lua b/test/functional/vimscript/let_spec.lua
index 4e95cd636..00dede799 100644
--- a/test/functional/vimscript/let_spec.lua
+++ b/test/functional/vimscript/let_spec.lua
@@ -28,7 +28,7 @@ describe(':let', function()
   it(":unlet self-referencing node in a List graph #6070", function()
     -- :unlet-ing a self-referencing List must not allow GC on indirectly
     -- referenced in-scope Lists. Before #6070 this caused use-after-free.
-    expect_exit(100, source, [=[
+    expect_exit(1000, source, [=[
       let [l1, l2] = [[], []]
       echo 'l1:' . id(l1)
       echo 'l2:' . id(l2)
